src/lib/Zikula/Composer/ManuallyInstallAssets.php

```diff
@@ -29,6 +29,12 @@ class ManuallyInstallAssets extends ScriptHandler
     private static $assets = [
         '/jQuery.mmenu/dist/js/jquery.mmenu.all.min.js' => '/jquery-mmenu/js/jquery.mmenu.all.min.js',
         '/jQuery.mmenu/dist/css/jquery.mmenu.all.css' => '/jquery-mmenu/css/jquery.mmenu.all.css',
+        '/twitter/typeahead.js/dist/bloodhound.js' => '/typeahead.js/bloodhound.js',
```

Comment:
Updated path in Route module template in 3f89807358dc0de12ff1fc1a0b837a4140c166e0